pkgs/top-level/all-packages.nix

```diff
@@ -438,6 +438,12 @@ with pkgs;
 
   catppuccin-plymouth = callPackage ../data/themes/catppuccin-plymouth { };
 
+  basalt-monado = callPackage ../by-name/ba/basalt-monado/package.nix {
+    tbb = tbb_2021_11;
+    cereal = cereal_1_3_2;
+    opencv = opencv.override{ enableGtk3 = true; };
+  };
+
```

Comment:
Packages in `by-name` are included by default without having to add them to `all-packages.nix`.

I'm pretty sure you should be overriding opencv in the derivation, *NOT* here.